src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import core.basesyntax.servise.FileReadService;
+import core.basesyntax.servise.FileWriteService;
+import core.basesyntax.servise.FruitService;
+import core.basesyntax.servise.ReadFile;
+import core.basesyntax.servise.WriteFile;
+import java.util.List;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    public static void main(String[] args) {
+        FileReadService readService = new ReadFile();
+        List<ProductsDto> lists = readService.readFromFile("src/test/resources/FruitTestOk.csv");
```

Comment:
```suggestion
        List<ProductsDto> productDtos = readService.readFromFile("src/test/resources/FruitTestOk.csv");
```

let's review variable and method names all over the project
so that they were more clear